codecov.yaml

```diff
@@ -0,0 +1,13 @@
+coverage:
+  status:
+    project:
+      default:
+        target: auto
+        # allow coverage to drop by this amount and still post success
+        threshold: 2%
```

Comment:
I think it suffices to limit the threshold to 0.5%, given the size of the code base.
Would it be possible to reduce the severity of a codecov failure (from red cross to yellow dot in github)?

Reply:
I wasn't able to find an option for the yellow dot in the configuration.  From my reading of the documentation I am not sure if this is 2% of the content of the existing pr, 2% of the overall code coverage, or both.  If is of the overall coverage than I agree with you that I could be much smaller.

Reply:
I guess it's the percentage reported by the codecov reports itself. What I have seen so far, are values like -0.39%. Maybe check some recent PRs to get a better impression of the actual range.

Reply:
changed it to 0.5